.gitignore

```diff
@@ -112,3 +112,5 @@ ENV/
 metadata.csv
 /data/
 db.sqlite3
+node_modules
+.DS_Store
```

Comment:
`cypress/.DS_Store` snuck into the PR despite this

Reply:
Oops. It _should_ be gone now.

Reply:
Oh, the original `.DS_Store` line in `.gitignore` was just fine. I think what happened was that when the `.gitignore` line was added, that was after `.DS_Store` was already `git add`ed, even if not yet `git commit`ed.